plutus-contract/src/Data/Text/Extras.hs

```diff
@@ -1,8 +1,17 @@
+{-# LANGUAGE OverloadedStrings #-}
+
 module Data.Text.Extras(
-    tshow
+      abbreviate
+    , tshow
     ) where
 
 import qualified Data.Text as T
 
 tshow :: Show a => a -> T.Text
 tshow = T.pack . show
+
+abbreviate :: Int -> T.Text -> T.Text
```

Comment:
The `abbreviate` function used to be in a `Utils` model in `plutus-pab` in which almost everything was unused so it seemed like a better fit to move it here instead.